src/sql/topics.sql

```diff
@@ -1,7 +1,7 @@
 DROP TABLE IF EXISTS `topics`;
 
 CREATE TABLE `topics` (
-  `topic_id` int(11) NOT NULL,
+  `topic_id` int(11) NOT NULL AUTO_INCREMENT,
```

Comment:
Database changes should not go here. They should go in the migrations directory.

Reply:
Turns out one of the migrations, `import_uwsolar_schema`, already makes the database change of making sure `topic_id` auto-increments.

https://github.com/kjiwa/lotus-leaf/blob/master/src/migration/alembic/versions/b80fb9e8acd7_import_uwsolar_schema.py#L33